src/Microsoft.SqlTools.Credentials/project.json

```diff
@@ -0,0 +1,135 @@
+{
+  "name": "Microsoft.SqlTools.Credentials",
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "emitEntryPoint": true,
+    "embed": [
+      "../Microsoft.SqlTools.ServiceLayer/Localization/sr.es.resx",
+      "../Microsoft.SqlTools.ServiceLayer/Localization/sr.resx"
+    ],    
+    "compile": {
+      "exclude": "*.*",
```

Comment:
If we're going the route of having this be its own standalone JSON RPC service, we need to make certain that we decouple it from the ServiceLayer project. I strongly encourage you to factor out the /Hosting/ folder into its own library that is shared between the two services. When I did the major refactor back in July of the code that came from the PowerShell dude, I had this exact scenario in mind.